apps/stats-web/package.json

```diff
@@ -24,7 +24,9 @@
     "geist": "^1.3.0",
     "iconoir-react": "^7.3.0",
     "jotai": "^2.5.1",
+    "lightweight-charts": "^4.2.0",
     "lucide-react": "^0.292.0",
+    "moment": "^2.30.1",
```

Comment:
Please remove moment and use `data-fns`, that's the library we use in other projects for time based functions.